src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Text;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DisplayNameTests
+    {
+        [Fact]
+        public void DisplayNameTest1()
```

Comment:
This is a great test. 
Can you do a similar one for AddColumns? That's interesting since the argument can't have a display name (since it's new and so doesn't map to the existing type).